Analysis/config/histograms_2016.cfg

```diff
@@ -0,0 +1,74 @@
+
+########## MVA & DNN BEGIN ##########
```

Comment:
why there are both 2016/histograms.cfg and histograms_2016.cfg? One of them should be removed.